scrabble.js

```diff
@@ -1,16 +1,189 @@
-
 const Scrabble = {
+  LETTERS: {
+    "a": { "points":  1, "tiles":  9 },
+    "b": { "points":  3, "tiles":  2 },
+    "c": { "points":  3, "tiles":  2 },
```

Comment:
I like that you've wrapped up point values and tile counts in one data structure - very clever!